### PR TITLE
Lesson3 home work anastasia goncear

### PR DESCRIPTION
**Pull request by AnastasiaGoncear**

### CODE REVIEW

File: src/main/java/Lesson3AnastasiaGoncear/Scanner.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package Lesson3AnastasiaGoncear;
+
+import java.io.InputStream;
+
+public class Scanner {
```
Comment (dgrimaliuc) [inline src/main/java/Lesson3AnastasiaGoncear/Scanner.java:5]: You accidentally created this class as I understand, 
Please remove it

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]

**Comment by AnastasiaGoncear** [comment]
Please check if the Scanner class is deleted. Still cannot see the current status of my PR.

File: src/main/java/Lesson3AnastasiaGoncear/ArraysForEach.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package Lesson3AnastasiaGoncear;
+
+public class ArraysForEach {public static void main(String[] args) {
+    // Example array
+    int[] array = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+
+    // Create a Scanner object for user input
+    Scanner scanner = new Scanner(System.in);
```
Comment (dgrimaliuc) [inline src/main/java/Lesson3AnastasiaGoncear/ArraysForEach.java:8]: import scanner

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]

File: src/main/java/Lesson3AnastasiaGoncear/ArraysForEach.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    // Prompt the user to enter the element to search for
+    System.out.print("Enter the element to search for: ");
+    int element = scanner.nextInt();
+
+    // Close the scanner
+    scanner.close();
```
Comment (dgrimaliuc) [inline src/main/java/Lesson3AnastasiaGoncear/ArraysForEach.java:15]: don't need to close it

**Review by dgrimaliuc — CHANGES_REQUESTED** [review]

**Review by dgrimaliuc-crunchyroll — APPROVED** [review]

**Review by dgrimaliuc — APPROVED** [review]

**Pull request merged**